### ISSUE TITLE
`json_decode()` exception

### ISSUE THREAD

**Issue by ike709** [wixoaGit/OpenDream#644]
```
[ERRO] world.log: Exception Occured: Invalid ValueKind Null
=DM StackTrace=
   json_decode(...)
   /datum/server_configuration/load_configuration(...)
   /world/New(...)

=C# StackTrace=
System.Exception: Invalid ValueKind Null
   at OpenDreamRuntime.Procs.Native.DreamProcNativeRoot.CreateValueFromJsonElement(JsonElement jsonElement) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\Procs\Native\DreamProcNativeRoot.cs:line 780
   at OpenDreamRuntime.Procs.Native.DreamProcNativeRoot.NativeProc_json_decode(DreamObject instance, DreamObject usr, DreamProcArguments arguments) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\Procs\Native\DreamProcNativeRoot.cs:line 828
   at OpenDreamRuntime.Procs.NativeProc.State.InternalResume() in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\Procs\NativeProc.cs:line 56
   at OpenDreamRuntime.ProcState.Resume() in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\DreamThread.cs:line 97
```
on https://github.com/ike709/Paradise/tree/od_yeet

**Comment by ike709** [comment]
Got this immediately before:
```
[ERRO] world.log: Exception Occured: Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
=DM StackTrace=
   /datum/configuration_section/event_configuration/(init)(...)
   <InitDreamObject DreamObject(/datum/configuration_section/event_configuration)>

   /datum/server_configuration/load_configuration(...)
   /world/New(...)

=C# StackTrace=
System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
   at OpenDreamRuntime.Objects.DreamList.SetValue(DreamValue key, DreamValue value) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\Objects\DreamList.cs:line 85
   at OpenDreamRuntime.Procs.DMOpcodeHandlers.ListAppendAssociated(DMProcState state) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\Procs\DMOpcodeHandlers.cs:line 210
   at OpenDreamRuntime.Procs.DMProcState.InternalResume() in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\Procs\DMProc.cs:line 196
   at OpenDreamRuntime.ProcState.Resume() in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\DreamThread.cs:line 97
   ```

**Comment by wixoaGit** [comment]
Fixed by ac210069db7a26bd5e2ddcc1c99414b12aa7d90c.
